setup.py

```diff
@@ -590,7 +590,9 @@ def get_exts_for(name):
             )
         )
 
-        ext_modules.append(CMakeExtension("ddtrace.appsec._iast._taint_tracking._native", source_dir=IAST_DIR))
+        ext_modules.append(
+            CMakeExtension("ddtrace.appsec._iast._taint_tracking._native", source_dir=IAST_DIR, optional=False)
```

Comment:
formatting might be off

Reply:
This is the code format/style that `hatch -v run lint:fmt .` prefers :sweat_smile: